### PR TITLE
Add ProfilePane with Avatar, Name, and UserName to ProfilePage Body

### PR DESCRIPTION
**Pull request by denisemauldin**
Fixes #292

### CODE REVIEW

**Comment by denisemauldin** [comment]
Screen Shot 2020-06-29 at 3 22 07 PM

File: README.md [hunk 1]
```diff
@@ -16,6 +20,10 @@ cd ../packages/neoFrontend
 npm start # Leave this open, it's a `create-react-app` dev server on port 3000
 ```
 
+To login, we use the Github integration with the development credentials below.
```
Comment (curran) [inline README.md:23]: Nice. Thanks for the documentation!

File: packages/neoFrontend/src/pages/ProfilePage/ProfilePane/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Wrapper, AuthorAvatar, AuthorInfo, AuthorName, AuthorUserName } from './styles';
+import { getUserName, getUserFullName } from 'vizhub-presenters';
+import { Avatar } from '../../../Avatar';
+
+export const ProfilePane = ({user, isSmall = false}) => {
+  console.log("User in profilePane", user)
```
Comment (curran) [inline packages/neoFrontend/src/pages/ProfilePage/ProfilePane/index.js]: Leftover `console.log`. I'll remove this.

**Review by curran — APPROVED** [review]
Looks fantastic!

**Pull request merged**